downstream/attributes/attributes.adoc

```diff
@@ -162,15 +162,13 @@
 :MenuAEAdminActivityStream: menu:Views[Activity Stream]
 :MenuAEAdminWorkflowApprovals: menu:Views[Workflow Approvals]
 :MenuAETemplates: menu:Resources[Templates]
-:MenuAMCredentials: menu:Resources[Credentials]
 :MenuAEProjects: menu:Resources[Projects]
 :MenuInfrastructureInventories: menu:Resoures[Inventories]
```

Comment:
Hey @dcdacosta, I noticed this attribute has "Resources" spelled wrong.

Reply:
@EMcWhinn that should be ok because it's now been deleted.